package.json

```diff
@@ -73,6 +73,7 @@
     "mini-css-extract-plugin": "^0.9.0",
     "npm-run-all": "^4.1.5",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
+    "polished": "^3.4.4",
```

Comment:
I've added `polished` (made by the same team as `styled-components`). Feel free to comment on this!

Reply:
Looks really useful! I think there are a few other existing places in the codebase where we could use things like the `rgba` helper.